src/ipsis/course-merge/CourseMergeMergedOfferingCollectionEntity.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * SelectedCourseMergeOfferingCollectionEntity class representation of a list of course merge offering as defined in the LMS
+ * See: ISirenCourseMergeSerializer.SerializeSelectedCourseOfferingListResult
+ */
+import { Actions, Classes } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class CourseMergeMergedOfferingCollectionEntity extends Entity {
+	originalSourceCourseMergeOfferings() {
+		if (!this._entity) {
+			return;
+		}
+		return this._entity.entities?.filter(course => !course.class.includes(Classes.ipsis.originalTarget));
+	}
+
+	originalTargetCourseMergeOffering() {
+		return this._entity?.getSubEntitiesByClass(Classes.ipsis.originalTarget)?.[0];
+	}
+
+	userOwnedByMultipleSourceSystems() {
+		return this._entity?.properties?.userOwnedByMultipleSourceSystems;
+	}
+
+	hasUnmergeAction() {
+		return this._entity.hasActionByName(Actions.ipsis.sisCourseMerge.unmergeCourseOfferings);
```

Comment:
This is on me, but if the other functions use optional chaining on `_entity?` maybe we should use it here and on line 34 too?